sdk/src/main/scala/com/horizen/account/receipt/EthereumReceipt.scala

```diff
@@ -130,7 +113,9 @@ object EthereumReceiptSerializer extends SparkzSerializer[EthereumReceipt] {
 
     // optional field
     val contractAddressLength = reader.getInt
-    val contractAddress: Option[Array[Byte]] = if(contractAddressLength == 0) None else Some(reader.getBytes(contractAddressLength))
+    val contractAddressBytes = reader.getBytes(contractAddressLength)
```

Comment:
honestly it is a bit misleading to read 0 bytes.
let's reformat to:
`if (contractAddressLength == 0) None else Some(new Address(reader.getBytes(contractAddressLength)))`